content/installation/installing-manager.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: bt_wiki
+title: Installing and Configuring Cloudify Manager
+category: Installation
+draft: false
+weight: 100
+---
+A Cloudify Manager is a compute host that runs the Cloudify Management service. For version 4.3 and higher, you can install a single RPM file that installs Cloudify Manager with all of its dependencies. The Cloudify Manager RPM file is self-contained and does not require an internet connection during installation.
+
+{{% gsNote title="Prerequisites" %}}
+
+Make sure that your environment meets the [prerequisites]({{< relref "installation/prerequisites.md" >}}) before you install Cloudify Manager.
+
+{{% /gsNote %}}
+
+When you install Cloudify Manager, you can specify the private IP address, public IP address and administrator password as command parameters. If you use _-v_ for the cfy_manager command, you can see installation logs located at: ```/var/log/cloudify/manager/cfy_manager.log```. You can also specify installation parameters in the [config.yaml file]({{< relref "installation/installing-manager.md#changing-the-manager-parameters" >}}).
```

Comment:
Regarding `-v`: you always see installation logs, but if you pass `-v` you see additional debug logs. Those logs are also _always_ available in /var/log...

Reply:
Change to:   * If you use ```-v``` for the cfy_manager command, you can see additional debugging logs located at: ```/var/log/cloudify/manager/cfy_manager.log```. 
Moved it to below the command.